install.sh

```diff
@@ -34,6 +34,8 @@ function usage() {
 	echo "          The proxy port to send collectd data to."
 	echo "    --overwrite_collectd_config"
 	echo "          Overwrite existing collectd configurations in /etc/collectd/"
+  echo "    --app_configure"
+  echo "          Launch the interactive application detection installer"
```

Comment:
I know this is a pedantic comment, but actually it's useful to say once:
It seems like the indentation in this file got a bit strange.  Maybe you ran some pep8 fixer? (though this is a .sh) file.  Is there any way you can make a pass to conform the indentation.  That will minimize the diff and help with the review.